projects/storefrontstyles/scss/components/index.scss

```diff
@@ -3,7 +3,6 @@
 @import './content/index';
 @import './product/index';
 @import './misc/index';
-@import './user/index';
```

Comment:
Regarding styles I think some things were forgotten when creating user lib.
eg. styles for close-account were not moved.

We should also remove some of them from `my-account` directory in storefrontstyles.

Could you evaluate what was forgotten and move it to user lib and then removed the duplicates from storefrontstyles?